### DISCUSSION TITLE
Solve the Silithus NPC problem before the content becomes irrelevant

### DISCUSSION THREAD

**Discussion by davegrier** [chromiecraft/chromiecraft#3511] [Suggestions]
Silithus NPCs that started AQ quests were removed in order to prevent AQ quests from being started, but this disrupted many other aspects of Silithus that are relevant to the current bracket and may be less so once AQ comes out, potentially leaving them insufficiently tested with real player experience. A few examples of the disruption:

- [Aendel Windspear](https://classic.wowhead.com/npc=15293/aendel-windspear) is unavailable to teach leatherworking patterns, some of which are pre-bis and still useful. 
- Vish Kozus is unavailable to turn in "[Wanted - Deathclasp, Terror of the Sands,](https://classic.wowhead.com/quest=8283/wanted-deathclasp-terror-of-the-sands)" which rewards a pre-bis resto shaman dagger. 
- [Bor Wildmane](https://classic.wowhead.com/npc=15306/bor-wildmane) is unavailable to collect Encrypted Twilight Texts for scrolls to create items to specify which abyssal lords to spawn. Some abyssal lord gear is still bis even after BWL. 

It's understandable that these sacrifices were made to prevent AQ quest issues, but if some alternate solution could be found to prevent those quests from being completed or accepted while allowing these NPCs to fulfill their other functions, it would draw more players to the zone to more thoroughly test the content before it becomes irrelevant, as well as improve quality of life for abyssal lord farming.

**Comment by minthes** [comment]
About Aendel Windspear, other professions (Tailoring, enchanting, blacksmithing) have access to their respective quartermasters to buy the recipes but for some reason the leatherworking one was removed.

**Comment by Nyeriah** [comment]
Aendel has been readded to the game. Bor is still subject of discussion
